Makefile

```diff
@@ -143,23 +140,55 @@ build-test-typecheck: build-python-wheels
 		--file ./test/docker-compose.typecheck-tests.yml
 
 .PHONY: build-test-integration
-build-test-integration: build-services
+build-test-integration: build
 	$(WITH_LOCAL_GRAPL_ENV) \
 	$(DOCKER_BUILDX_BAKE) --file ./test/docker-compose.integration-tests.yml
 
 .PHONY: build-test-e2e
-build-test-e2e: build-services
+build-test-e2e: build
 	$(WITH_LOCAL_GRAPL_ENV) \
 	$(DOCKER_BUILDX_BAKE) --file ./test/docker-compose.e2e-tests.yml
 
+.PHONY: build-lambda-zips
+build-lambda-zips: build-lambda-zips-rust build-lambda-zips-js build-lambda-zips-python ## Generate all lambda zip files
+
+.PHONY: build-lambda-zips-rust
+build-lambda-zips-rust: ## Build Rust lambda zips
+	$(DOCKER_BUILDX_BAKE) --file docker-compose.lambda-zips.rust.yml
+	# Extract the zip from the Docker image.
+	# Rely on the default CMD for copying artifact to /dist mount point.
+	docker run \
+		--rm \
+		--user "${UID}:${GID}" \
+		--mount type=bind,source="${PWD}/dist",target=/dist \
+		grapl/metric-forwarder-zip:${TAG}
+
+.PHONY: build-lambda-zips-js
+build-lambda-zips-js: ## Build JS lambda zips
+	$(DOCKER_BUILDX_BAKE) --file docker-compose.lambda-zips.js.yml
+	# Extract the zip from the Docker image.
+	# Rely on the default CMD for copying artifact to /dist mount point.
+	docker run \
+		--rm \
+		--user "${UID}:${GID}" \
+		--mount type=bind,source="${PWD}/dist",target=/dist \
+		grapl/graphql-endpoint-zip:${TAG}
```

Comment:
We could use `docker-compose run` here instead, which IMO has a benefit of anchoring on the image name defined in the compose file, so if the image name changes in that file you wouldn't have to update it here as well. I chose `docker run` just to avoid compose altogether, but I think we could try using `docker-compose run` and see if it gives us issues. It's still better than using compose up, which was previously the case.

Reply:
I guess I'm agnostic on this point. It's definitely better than `docker-compose up`. Having the image name "take care of" in the compose file _is_ nice.

On the other hand, moving away from Docker Compose for things that aren't actually services is _also_ nice. This could potentially open the door to moving our build-only compose files to HCL for `buildx bake`, which solidifies that distinction. However, that's just speculation at this point, and wouldn't be hard to implement either way.

I'm fine with `docker run` 😄

Reply:
Yeah.. I'm tempted to switch back to 'docker-compose run' for now, not just for handing the image name, but also because 'docker-compose run' is used elsewhere in the Makefile, so it'd also make Makefile more consistent.

I think I'll make the switch, since you seem fine either way, though I'll ping you for confirmation before merging to main. I see this has been accepted already (ty ty).

Reply:
(such as `lint-prettier`)